src/provider/provider_level_zero.c

```diff
@@ -201,6 +201,9 @@ static umf_result_t ze2umf_result(ze_result_t result) {
     }
 }
 
+#define ZE_CALL(ZeName, ZeArgs)                                                \
+    (LOG_INFO("ZE ---> %s%s", #ZeName, #ZeArgs), g_ze_ops.ZeName ZeArgs)
```

Comment:
Do you think it should be `INFO` not `DEBUG`? BTW, I have no strong opinion.

Reply:
DEBUG is pretty verbose right now (e.g. all inserts to memory tracker are printed). This functionality (tracing all calls to L0) is often used for debugging SYCL programs. Having all this debug information from UMF would probably be too much.